src/python/grpcio/grpc/_channel.py

```diff
@@ -652,16 +663,24 @@ def __init__(self, channel):
     self.managed_calls = None
 
 
-def _call_spin(state):
-  while True:
-    event = state.completion_queue.poll()
-    completed_call = event.tag(event)
-    if completed_call is not None:
-      with state.lock:
-        state.managed_calls.remove(completed_call)
-        if not state.managed_calls:
-          state.managed_calls = None
-          return
+def _run_call_spin_thread(state):
```

Comment:
`_run_channel_spin_thread`, please, since there will be one spin thread (at most) per channel, not per call.